From 36192a6755783c7c74ddfe934e22bd64f5d4fd64 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Fri, 16 Jun 2023 10:59:52 +0530
Subject: [PATCH] Atrace service is deprecated in Android 14

Atrace HAL is depcrecated in Target 8 and API 34
and should not be serviced.

Tracked-On: OAM-110592
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 caas/mixins.spec     | 2 +-
 caas_dev/mixins.spec | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/caas/mixins.spec b/caas/mixins.spec
index 0966dc2..eab2414 100755
--- a/caas/mixins.spec
+++ b/caas/mixins.spec
@@ -81,7 +81,7 @@ load_modules: true
 gptbuild: true(size=32G,generate_craff=false,compress_gptimage=true)
 dynamic-partitions: true(super_img_in_flashzip=true,super_partition_size=5000,virtual_ab=true,virtual_ab_compression=true)
 dbc: true
-atrace: true
+atrace: false
 firmware: true(all_firmwares=false)
 aaf: true
 suspend: auto
diff --git a/caas_dev/mixins.spec b/caas_dev/mixins.spec
index 5c8f0b5..6d24263 100755
--- a/caas_dev/mixins.spec
+++ b/caas_dev/mixins.spec
@@ -81,7 +81,7 @@ load_modules: true
 gptbuild: true(size=6G,generate_craff=false,compress_gptimage=true)
 dynamic-partitions: true(super_img_in_flashzip=true,super_partition_size=5120,virtual_ab=true,virtual_ab_compression=true)
 dbc: true
-atrace: true
+atrace: false
 firmware: true(all_firmwares=false)
 aaf: true
 suspend: auto
-- 
2.17.1

